prow/lib.sh

```diff
@@ -125,11 +128,13 @@ function build_images() {
 # Creates a local registry for kind nodes to pull images from. Expects that the "kind" network already exists.
 function setup_kind_registry() {
   # create a registry container if it not running already
+  registry_img="gcr.io/istio-testing/registry:2"
```

Comment:
same

Reply:
gcr.io/istio-testing/registry:2.8.0

Reply:
Yes, it seems multi-arched for "gcr.io/istio-testing/registry:2.8.0"
If all images used in CI are multi-arched, it would be of great help

Reply:
can you update it to 2.8.0?

Reply:
OK

Reply:
Updated.